Docs/source/developers/particles.rst

```diff
@@ -141,7 +141,7 @@ Attribute name        ``int``/``real``  Description                         Wher
                                         Wheeler process                                physics is used.
 ====================  ================  ==================================  ===== ==== ======================
 
-WarpX allows extra runtime attributes to be added to particle containers (through ``NewRealComp("attrname")`` or ``NewIntComp("attrname")``).
+WarpX allows extra runtime attributes to be added to particle containers (through ``AddRealComp("attrname")`` or ``AddIntComp("attrname")``).
```

Comment:
Most likely break that happens in user code:
`NewRealComp` -> `AddRealComp`
`NewIntComp` -> `AddIntComp`